### DISCUSSION TITLE
Hasura Data Connectors vs Remote Schemas

### DISCUSSION THREAD

**Discussion by abaltatu** [hasura/graphql-engine#9359] [Question]
Hello. 

I would like to know when should one choose Hasura Data Connectors versus Remote Schemas? They seem pretty similar in that they both are standalone servers that provide the engine with schemas and have GraphQL resolvers. And both can connect various data sources. 

Thank you!

**Comment by adas98012** [comment]
@abaltatu 
Remote Schemas are only applicable for custom **GraphQL** schemas whereas Data Connectors are for a niche group of Data sources (like Amazon Athena and Snowflake).
